subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -231,6 +284,19 @@ private LinePrefixingStyledTextOutput getLinePrefixingStyledTextOutput(FailureDe
 
             return new LinePrefixingStyledTextOutput(details.details, prefix, false);
         }
+
+        @Override
+        protected void endVisiting() {
+            if (suppressedDuplicateBranchCount > 0) {
+                LinePrefixingStyledTextOutput output = getLinePrefixingStyledTextOutput(failureDetails);
+                boolean plural = suppressedDuplicateBranchCount > 1;
+                if (plural) {
+                    output.append(String.format("There were %d additional failures with the same cause that were not printed.", suppressedDuplicateBranchCount));
```

Comment:
WDYT?
```suggestion
                    output.append(String.format("There are %d more failures with identical causes.", suppressedDuplicateBranchCount));
```

Reply:
I'll adjust the wording on this (and singular case to match) tomorrow.